atomic_reactor/plugins/prepub_squash.py

```diff
@@ -64,7 +62,7 @@ def __init__(self, tasker, workflow, tag=None, from_base=True, from_layer=None,
         :param from_base: bool, squash from base-image layer, on by default
         :param from_layer: layer from we will squash - if specified, takes precedence over from_base
         :param tag: str, new name of the image - by default use the former one
-        :param remove_former_image: bool, remove unsquashed image?
+        :param remove_former_image: not used
```

Comment:
You can remove that parameter from the docstring and also from the constructor's signature... like you had before I asked you to change it (sorry).